### PR TITLE
chore: Remove prisma from the Builder app

### PR DESCRIPTION
**Pull request by istarkov**
## Description

1. What is this PR about (link the issue and add a short description)

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/app/routes/rest.current.products.ts [hunk 1]
```diff
@@ -1,71 +0,0 @@
-import { json, type LoaderFunctionArgs } from "@remix-run/server-runtime";
```
Comment (TrySound) [inline apps/builder/app/routes/rest.current.products.ts:1]: We don't use it in n8n?

File: apps/builder/app/shared/$resources/sitemap.xml.server.ts [hunk 2]
```diff
@@ -22,25 +22,24 @@ export const loader = async ({ request }: { request: Request }) => {
...
-  if (build === null) {
-    throw json({ message: "Build not found" }, { status: 404 });
+  const context = await createContext(request);
+
+  const buildResult = await context.postgrest.client
+    .from("Build")
+    .select("pages, updatedAt")
+    .eq("projectId", projectId)
+    .is("deployment", null)
+    .single();
```
Comment (TrySound) [inline apps/builder/app/shared/$resources/sitemap.xml.server.ts:32]: pages can be big, no perf penalty?

**Review by TrySound — APPROVED** [review]
Love it!

File: apps/builder/app/routes/rest.current.products.ts (hunk 1 above)
Comment (istarkov) [reply apps/builder/app/routes/rest.current.products.ts:1]: decided to trust this comment
/**
 * Created for ebugging purposes, to check that payments and products are working
 * Showing current user's checkouts with products.
 */

File: apps/builder/app/shared/$resources/sitemap.xml.server.ts (hunk 2 above)
Comment (istarkov) [reply apps/builder/app/shared/$resources/sitemap.xml.server.ts:32]: Already fixed on pg side

**Pull request merged**